spec/mongo/operation/write/command/insert_spec.rb

```diff
@@ -56,6 +56,31 @@
     end
   end
 
+  describe 'write concern' do
+
+    context 'when write concern is not specified' do
+
+      let(:spec) do
+        { :documents     => documents,
+          :db_name       => db_name,
+          :coll_name     => coll_name,
+          :ordered       => true
+        }
+      end
+
+      it 'does not include write concern in the selector' do
+        expect(op.send(:selector)[:writeConcern]).to be_nil
```

Comment:
Should the write concern be `nil` or `{}` here? The spec makes it look like it should be the server's default write concern, which appears to be `{}` and not `nil`. Maybe verify this with craig?

https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#servers-default-writeconcern